l10n/en/privacy/firefox.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/privacy/firefox
+
+privacy-firefox-mozilla-privacy = { -brand-name-mozilla } Privacy
```

Comment:
Shouldn't this be Firefox (vs Mozilla)?

Reply:
It's a link to the Mozilla privacy policy. I'll re-organize the file to put it with the other sidebar links.